packages/core/src/modules/oob/OutOfBandService.ts

```diff
@@ -28,6 +38,43 @@ export class OutOfBandService {
     this.eventEmitter = eventEmitter
   }
 
+  public async createImplicitInvitation(
+    agentContext: AgentContext,
+    config: CreateImplicitInvitationConfig
+  ): Promise<OutOfBandRecord> {
+    const { did, threadId, handshakeProtocols, autoAcceptConnection, recipientKey } = config
+
+    // Verify it is a valid did and it is present in the wallet
+    const publicDid = parseDid(did)
+    if (!(await agentContext.dependencyManager.resolve(DidsApi).getCreatedDids({ did: publicDid.did }))) {
```

Comment:
getCreatedDids will always return an array I think?

```suggestion
    const didsApi = agentContext.dependencyManager.resolve(DidsApi)
    const [createdDid] = await didsApi.getCreatedDids({ did: publicDid.did })
    if (!createdDid) {
```